scripts/production/run_healthchecks.pl

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env perl
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
```

Comment:
Not sure if in scripts `=head1 LICENSE` should be added between the shebang line and the copyright.

Reply:
The convention for scripts is that the license should not be in a `-head` section but in `#` comments. See https://www.ebi.ac.uk/seqdb/confluence/display/ENS/Ensembl+Software+License